.travis.yml

```diff
@@ -15,8 +15,6 @@
 
 language: java
 
-sudo: false
```

Comment:
@bzz This is just for testing. I am struggling to verify why livy integration test fails on travis CI (I can run in local successfully). Sometimes the livy server will be killed unexpectedly, do you have any clues for that ?
```
 INFO [2016-12-06 14:36:02,147] ({main} LivyInterpreterIT.java[onUpdate]:303) - onUpdate:
DEBUG [2016-12-06 14:36:02,148] ({main} LivyHelper.java[executeHTTP]:346) - Call rest api in http://testing-docker-ca2e7f34-da95-4627-bc72-7e92273f5758:8998/sessions/0/statements, method: POST, jsonData: {"code": "sqlContext.sql(\"show tables\").show(100)"}
DEBUG [2016-12-06 14:36:03,274] ({main} LivyHelper.java[executeHTTP]:346) - Call rest api in http://testing-docker-ca2e7f34-da95-4627-bc72-7e92273f5758:8998/sessions/0/statements/4, method: GET, jsonData: null
DEBUG [2016-12-06 14:36:03,392] ({main} LivyHelper.java[getStatusById]:323) - statement 4 response: {"id":4,"state":"running","output":null}
DEBUG [2016-12-06 14:36:04,392] ({main} LivyHelper.java[executeHTTP]:346) - Call rest api in http://testing-docker-ca2e7f34-da95-4627-bc72-7e92273f5758:8998/sessions/0/statements/4, method: GET, jsonData: null
DEBUG [2016-12-06 14:36:04,528] ({main} LivyHelper.java[getStatusById]:323) - statement 4 response: {"id":4,"state":"running","output":null}
DEBUG [2016-12-06 14:36:05,529] ({main} LivyHelper.java[executeHTTP]:346) - Call rest api in http://testing-docker-ca2e7f34-da95-4627-bc72-7e92273f5758:8998/sessions/0/statements/4, method: GET, jsonData: null
/home/travis/build.sh: line 57:  6800 Killed                  mvn $TEST_FLAG $PROFILE -B $TEST_PROJECTS
 
 
The command "mvn $TEST_FLAG $PROFILE -B $TEST_PROJECTS" exited with 137.
store build cache
 
change detected (content changed, file is created, or file is deleted):
/home/travis/.m2/repository/org/jsoup/jsoup/1.9.3-SNAPSHOT/resolver-status.properties
/home/travis/.m2/repository/org/jsoup/jsoup/resolver-status.properties
 
 
changes detected, packing new archive
```